data/transactions/logic/eval_test.go

```diff
@@ -1815,6 +1831,7 @@ int 2
 			}
 			ep := defaultEvalParams(nil, &txn)
 			ep.TxnGroup = makeSampleTxnGroup(txn)
+			ep.Ledger = makeTestLedger(nil)
```

Comment:
I'm a little worried now.  If you had to add the ledger here, I don't know what's going to happen when this is used in a logicsig.  I don't know that the ledger is available there.  Do you have an e2e with a logicsig using this?